app/javascript/stylesheets/application.scss

```diff
@@ -165,6 +165,7 @@ header.navbar {
   grid-row-gap: .5rem;
   grid-column-gap: .3rem;
   grid-template-columns: .3fr 1fr 1fr 1fr 1fr;
```

Comment:
There are two `grid-template-columns` declarations. I think we only need the second (newer) one.

Reply:
yeah, my editor's formatter formatted the whole file and I didn't want to commit all the changes, because I don't know if that formatting is OK. Apparently I selected too much for commit :)

Reply:
fixed

Reply:
Thank you!